wordpress/README.md

```diff
@@ -0,0 +1,280 @@
+Mon Apr 27 00:51:36 UTC Generating the cluster image
+Mon Apr 27 00:51:44 UTC Saving the image as wordpress.tar
+        Still saving the image as wordpress.tar (10 seconds elapsed)
+Mon Apr 27 00:52:02 UTC Build finished in 3 minutes 
+```
+
+Let's review what just happened. `tele build` did the following:
+
+* Downloaded Kubernetes binaries and Gravity tooling from `s3://hub.gravitational.io`.
+* Scanned the current directory and the subdirectories for Kubernetes resources and Helm charts.
+* Downloaded external container images referenced in the resources discovered in the previous step.
+* Packaged (or vendored) Docker images into the Cluster Image.
+* Removed the duplicate container image layers, reducing the size of the Cluster Image.
+* Saved the Cluster Image as `wordpress.tar`.
+
+Note: Slow Operation Warning
+    `tele build` needs to download hundreds of megabytes of binary dependencies which
+    can take a considerable amount of time, depending on your Internet connection speed.
+
+The resulting `wordpress.tar` file is about 2.9GB and it is **entirely
+self-sufficient** and dependency-free. It contains everything: the Kubernetes
+binaries, the Docker engine, the Docker registry and the Wordpress application itself:
+everything one needs to get Wordpress up and running on any fleet of Linux
+servers (or into an AWS/GCE/Azure account).
+
+Congratulations! You have created your first **Kubernetes virtual appliance**!
+
+## Installing
+
+Installing the `wordpress.tar` Cluster Image results in creating a Kubernetes
+cluster with the application pre-loaded. This file is the only artifact
+one needs to create a Kubernetes cluster with Wordpress running inside.
+
+Copy `wordpress.tar` to a clean Linux machine. Let's call it `host`. This node
+will be used to bootstrap the cluster. Let's untar it and look inside:
+
+```bash
+$ tar -xf wordpress.tar
+$ tree
+├── app.yaml
+├── gravity
+├── gravity.db
+├── install
+├── packages
+│   ├── blobs
+│   │   ├── 0e1
+│   │   ...
+│   │   └── ff1
+│   │       └── ff19bcf2dc62f037e0016d5d065150d195f714be3c97a301791365a4ec5a43f0
+│   ├── tmp
+│   └── unpacked
+├── README
+├── upgrade
+└── upload
+```
+
+
+### Installing via CLI
+
+To install a Cluster via CLI, you have to execute the `./gravity install` command and
+supply three flags:
+
+Flag              | Description
+-------------------|---------------------------------
+`--token`          | A secret token of your choosing which will be used to add additional nodes to this Cluster in the future. We'll use word "secret" here.
+`--advertise-addr` | The IP address this host will be visible on by other nodes in this Cluster. We'll use `10.5.5.28`.
+`--cloud-provider` |  Whether this is a generic, aws or gcp installation.  We'll use `generic`.
+
+The command below will create a single-node Kubernetes cluster with Wordpress running inside:
+
+```
+# We are executing this on the node named 'host' with IP address of 10.5.5.28
+$ sudo ./gravity install \
```

Comment:
Your description above mentions that `It can be just one machine but this example quickstart will use three.` but it actually only seems to install a single-node?

Reply:
Modified to one linux node.  A note at the bottom details the flavor option.  The docs version will be more detailed that can use 3 nodes.